package_requirements.txt

```diff
@@ -0,0 +1,2 @@
+build==0.7.0
+twine==4.0.0
```

Comment:
is this replacing the current `requirements.txt`? Do we need to move over all the dependencies originally in there?